systemd/openqa-webui.service

```diff
@@ -6,10 +6,9 @@ After=postgresql.service openqa-setup-db.service openqa-scheduler.service nss-lo
 Requires=openqa-livehandler.service openqa-websockets.service openqa-gru.service openqa-enqueue-asset-cleanup.timer openqa-enqueue-result-cleanup.timer openqa-enqueue-bug-cleanup.timer
 
 [Service]
-# TODO: define whether we want to run the web ui with the same user
 User=geekotest
-# Our API commands are very expensive, so the default timeouts are too tight
-ExecStart=/usr/share/openqa/script/openqa prefork -m production --proxy -i 100 -H 400 -w 30 -c 1 -G 800
+;ExecStart=/usr/share/openqa/script/openqa-webui-daemon
```

Comment:
?

Reply:
exactly, I wanted to show you how I tested, right? :) Fixed